src/lib/orm/Doctrine.php

```diff
@@ -62,18 +60,17 @@ private function __construct()
         }
 
         $conf = Config::getConf();
-        $dbUser = $conf->getDbUser();
-        $dbPass = $conf->getDbPass();
-        $dbHost = $conf->getDbHost();
-        $dbPort = $conf->getDbPort();
-        $dbName = $conf->getDbName();
-        $conn = [
+        $connectionParams = [
+            'dbname' => $conf->getDbName(),
+            'user' => $conf->getDbUser(),
+            'password' => $conf->getDbPass(),
+            'host' => $conf->getDbHost(),
+            'port' => $conf->getDbPort(),
             'driver' => 'pdo_mysql',
-            'url' => "mysql://$dbUser:$dbPass@$dbHost:$dbPort/$dbName",
```

Comment:
Fix for:
```log
Fatal error: Uncaught Doctrine\DBAL\Exception: Malformed parameter "url". in src/vendor/doctrine/dbal/src/DriverManager.php:306
```

Quick fix:
Inside [Doctrine.zip](https://github.com/orangehrm/orangehrm/files/9858772/Doctrine.zip), there is a PHP file `Doctrine.php`.
Find current `Doctrine.php` inside `<orangehrm>/src/lib/orm/` directory. And replace it with new file which is taken from the ZIP.